gluster-exporter/metric_ps.go

```diff
@@ -177,7 +175,7 @@ func ps() {
 		}
 		cmdlineArgs, err := getCmdLine(lineData[0])
 		if err != nil {
-			// TODO: Log error
+			log.WithError(err).Error("Error getting command line")
```

Comment:
Include pid and command name in log fields

Reply:
Ack